### ISSUE TITLE
Adding qBitTorrent as download client for Prowlarr failing

### ISSUE THREAD

**Issue by imgios** [fonzdm/servarr#22]
### Description

There's an issue when adding qBitTorrent in Prowlarr as download client via the API `ttp://servarr-prowlarr.{{ .Release.Namespace }}.svc.cluster.local:9696/api/v1/downloadclient\`. Moreover, Prowlarr is answering with the status code 400 and the following message: `Test was aborted due to an error: Object reference not set to an instance of an object.`.

From [Prowlarr docs](https://prowlarr.com/docs/api/#/DownloadClient/post_api_v1_downloadclient), the request body seems quite different from the one being set in the job:

https://github.com/fonzdm/servarr/blob/810bb482935e99e4a9430e767e3801473a99bdef/servarr/templates/init-downloaders.yaml#L105-L118

### Steps to reproduce

Deploy the Helm Chart with any value by using the version available in the `dev` branch.

### Expected behavior

qBitTorrent getting registered in Prowlarr successfully I guess?

### Servarr Helm Chart version

dev

### Relevant log output

```shell
# API Answer
~$ kubectl logs -n servarr downloaders-post-install-job-mpj59 -c prowlarr-vs-qbt-finalizer                                                                                                                         % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
[
  {
    "isWarning": false,
    "propertyName": "",
    "errorMessage": "Test was aborted due to an error: Object reference not set to an instance of an object.",
    "severity": "error"
  }
100  1033    0   194  100   839    539   2333 --:--:-- --:--:-- --:--:--  2869
]

# Prowlarr pod logs
[Error] QBittorrent: Test aborted due to exception

[v1.14.0.4286] System.NullReferenceException: Object reference not set to an instance of an object.
   at NzbDrone.Core.Download.DownloadClientBase`1.ValidateCategories(List`1 failures) in ./Prowlarr.Core/Download/DownloadClientBase.cs:line 97
   at NzbDrone.Core.Download.DownloadClientBase`1.Test() in ./Prowlarr.Core/Download/DownloadClientBase.cs:line 112


[Warn] ProwlarrErrorPipeline: Invalid request Validation failed:
 -- : Test was aborted due to an error: Object reference not set to an instance of an object.
```

### Chart values

```yaml
# Refers to https://github.com/fonzdm/servarr/blob/dev/.github/ci/ci-values.yaml
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fonzdm** [comment]
Hi @imgios, thanks for reporting this bug!
I will take care of it, let's see what's wrong. 
Thank you also for pointing the correct API to be used 😃 

I will create a dedicated branch for this, and i (hopefully) fix the bug using the current workflow (one container inside the job, executing the curl).

**Comment by fonzdm** [comment]
PR #23 has been raised.

**Comment by imgios** [comment]
Issue fixed in #23.
